tests/pom.xml

```diff
@@ -54,12 +54,6 @@
             <classifier>${cuda.version}</classifier>
             <scope>provided</scope>
         </dependency>
-        <dependency>
```

Comment:
2022 copyrights, and this seems like an unrelated change?

Reply:
Commit time is before 2021-01-01, so did not add 2022 copyright.